ios/NativeBridge.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  NativeBridge.swift
+//  A0Auth0
+//
+//  Created by Poovamraj T T on 15/06/23.
+//  Copyright © 2023 Facebook. All rights reserved.
+//
+
+import Auth0
+import Foundation
+import LocalAuthentication
+
+@objc
+public class NativeBridge: NSObject {
+    
+    
+    static let accessTokenKey = "accessToken";
+    static let idTokenKey = "idToken";
+    static let expiresAtKey = "expiresAt";
+    static let scopeKey = "scope";
+    static let refreshTokenKey = "refreshToken";
+    static let typeKey = "type";
+    static let tokenTypeKey = "tokenType";
+    static let expiresInKey = "expiresIn";
+    static let dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+    
+    static let errorCode = "a0.invalid_state.credential_manager_exception"
+    
+    var credentialsManager: CredentialsManager
+    var clientId: String
+    var domain: String
+    
+    @objc public init(clientId: String, domain: String) {
+        let auth0 = Auth0
+            .authentication(clientId: clientId, domain: domain)
+        self.clientId = clientId
+        self.domain = domain
+        self.credentialsManager = CredentialsManager(authentication: auth0)
+        super.init()
+   }
+    
+    @objc public func webAuth(scheme: String?, state: String?, nonce: String?, audience: String?, scope: String?, connection: String?, maxAge: Int, organization: String?, invitationUrl: String?, leeway: Int, ephemeralSession: Bool, additionalParameters: [String: String], resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
```

Comment:
Is `scheme` needed here?

Reply:
The method signature for iOS and Android has to be same. So we are adding the missing parameters as well.

Reply:
But this is the Swift code, that gets manually called from the ObjC code. Does it still need to have the same signature?